docs/root/index.rst

```diff
@@ -19,4 +19,5 @@ Envoy documentation
   operations/operations
   extending/extending
   api-v2/api
+  api-docs/xds_protocol
```

Comment:
I think we should have a single top-level "API", and nested under this the xDS wire and API reference entries.

Reply:
done